TFlippy_TerritoryControl_Structures_Dev/Entities/Structures/CargoContainer/CargoContainer.cfg

```diff
@@ -54,9 +54,9 @@ $attachment_factory                               = box2d_attachment
 @$attachment_scripts                              =
 @$attachment_points                               = PICKUP; -1; 4; 1; 0; 0;
 													CARGO; 0; -10; 1; 0; 0;	
-													PASSENGER0; -12; 0; 0; 0; 8;
-													PASSENGER1; 4; 0; 0; 0; 8;
-													PASSENGER2; 20; 0; 0; 0; 8;
+													PASSENGER; -12; 0; 0; 0; 8;
```

Comment:
Does this actually work..? Id assume you are not allowed multiple attachment points with the same name